app/models/concerns/audit_search.rb

```diff
@@ -15,7 +15,7 @@ module AuditSearch
     belongs_to :search_nics, -> { where('audits.auditable_type LIKE ?', "Nic::%") }, :class_name => 'Nic::Base', :foreign_key => :auditable_id
     belongs_to :search_settings, :class_name => 'Setting', :foreign_key => :auditable_id
 
-    scoped_search :on => [:username, :remote_address], :complete_value => true
+    scoped_search :on => [:username, :remote_address, :comment], :complete_value => true
```

Comment:
do you want full text search on comment field?  (e.g. when an user type in search "bla" it would also search inside the comment field).

since its the same table its probably not too bad to keep, but my default would be to do it explicitly.

Reply:
Full text search would be best, is there an additional option for that?

TBH, I'm not sure how useful comment field is since the db has it as 255 char instead of text. That makes for a very short comment in real usage. Better than nothing, though. :)